src/less/globals/themes/_techne.less

```diff
@@ -151,6 +151,17 @@
 @table-cell-select-color-active: #fff;
 @table-cell-select-color-hover: @default-blue;
 
+//== Trees
+
+//** Headers text color
+@tablehead-text-color: @default-greyblue;
+
+//** Rows text-color
+@tablerow-text-color: @dark-blue;
+
+//** Primary data for on focus rows text
+@primarydata-on-focus-text-color: @default-blue;
+
```

Comment:
can you please also add these vars to src/less/globals/themes/_belize.less.

we need to maintail a 1:1 vars with these two files as to not miss anything when we eventually update the belize theme.